kick/client.py

```diff
@@ -329,7 +329,18 @@ async def on_livestream_start(self, livestream: PartialLivestream) -> None:
         livestream: `PartialLivestream`
             The livestream
         """
+    async def on_livestream_end(self, StreamEnd: Endstream) -> None:
+        """
+        |coro|
+
+        on_livestream_start is an event that can be overriden with the `Client.event` decorator or with a subclass.
+        This is called when a user that is being watched starts streaming
 
+        Parameters
+        -----------
+        livestream: `PartialLivestream`
```

Comment:
These are the wrong parameters, appears you forgot to change those too when copy pasting the docstring